data/hai/hai reveal 5 philosophers.txt

```diff
@@ -416,18 +488,34 @@ mission "Hai Reveal [B06-long] Tarazed Analysis"
 			`	You clarify that technically it's a lead on cases that match the same profile as his wife's, not strictly his wife's case. His expression falters momentarily and then he seems to find some internal resolve to draw upon. "Good enough for me... Thank you, for your honesty."`
 			label lab
 			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, you realize it must be one of Tarazed's research shipyards. Goraz's laboratory is an entire docking bay with a modified Blackbird. Despite its size, it is conspicuously absent any other personnel. He takes you on board and, noting your attention to your surroundings, explains that he has few assistants that are typically off-world most of the time as he walks you to the communications room.`
```

Comment:
```suggestion
			`	After a short return trip, you reach what looks like a small private spaceport. From the extra equipment and unusual ship designs, you realize it must be one of Tarazed's research shipyards. Goraz's laboratory is an entire docking bay with a modified Blackbird, but despite its size, it is conspicuously absent any other personnel. He takes you on board and leads you to the communications room; noting your attention to your surroundings, Goraz explains that he has a few assistants, but they are off-world most of the time.`
```
This is still a little clunky I think.

Reply:
I made lots of changes to that section, and I think I incorporated everything you suggested.